torch/csrc/jit/passes/quantization.h

```diff
@@ -90,6 +90,8 @@ TORCH_API script::Module InsertQuantDeQuant(
  */
 TORCH_API void ReplicateDeQuant(std::shared_ptr<Graph>& graph);
 
+TORCH_API void SwapDeQuant(std::shared_ptr<Graph>& graph);
```

Comment:
Where is this API called? Shouldnt there be a corresponding change to convert_script?

Reply:
this should be called in convert_script, but changes to convert_script is blocked on freezing. We'll add these later

Reply:
What is the dependency? We should be able to call inline and this pass without depending on freezing isnt it?

Reply:
We don't have a `_finalize_quant` API right now, and needs to call make attributes constant that's provided in freezing. I think we can add that later when we change convert_script to be more close to the new planed API, doesn't have to change everything in one PR.